modules/ROOT/pages/config-add-proxy-whitelist.adoc

```diff
@@ -3,27 +3,36 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-By default, proxy requests are enabled for the domain name where the platform is running.
-
-To allow addresses:
-
-. Log in to Ops Center and select *Kubernetes* on the left side-bar.
-. Select the `api-manager` namespace from the drop-down menu on the top, and then select *Configmaps*. Find the `api-platform-web-env` configmap and then click on `Edit Config Map`.
-. Select the *APW_FEATURES_PROXYWHITELIST* tab and ensure that value is set to `true`. (You'll have to hover your mouse over the tabs to get a tooltip with the full name to find the right tab)
-+
-If not, set it to `true` and then select `Apply`.
-. Select the *APW_PROXYWHITELIST_RULES* tab.
-. Write your rules separated by commas.
-+
-The following example defines regular expressions that allow requests to be made to the `*.somewhere.com/*` and `*.somewhereelse.com/*` domains, where `*` is any part of a DNS name or URL:
-
-----
-.*\.somewhere\.com,.*\.somewhereelse\.com
-----
-
-. Select *Apply* to save changes to the `api-platform-web-env` config map.
-. Re-create the pods to ensure that each node in the cluster uses the most recent configuration:
-+
-----
-kubectl delete pod -n api-manager -l component=api-platform-web
-----
+== Registered Domains
+
+Every HTTP request you make to an external resource goes through the API Console Proxy. For security reasons, you need to register the domain as an allowed or blocked domain in Anypoint Platform. 
+
+By default, proxy requests are blocked for new domains on authenticated requests.
+
+To add a new domain:
+
+1. Log in to Anypoint Platform using an account that has the Organization Administrator permission.
+
+2. In the navigation bar or the main Anypoint Platform page, click *Access Management*.
+
+3. Click *Registered domains*.
+
+4. Click *Add domain*. +
+The domain must not match with any existing domain, and must not be any domain or subdomain of MuleSoft or Anypoint Platform.
+
+5. Under Trust level, select if you want the domain to be *Allowed* or *Blocked*.
+
+6. In the dialog box that appears, type the domain name and click Add domain. +
+
+Result: The domain has been added to your list of registered domains. You are redirected to the registered domains page, where you can edit or delete your domains. If you register a domain on a parent organization, it is also allowlisted on all your business groups.
```

Comment:
We should change this whole paragraph for the following: `Result: The domain has been added to your trusted domains. You will be able to see it listed on the trusted domains page, where you can also delete your domains. If you register a domain on a parent organization, it will be also allowed on all your child business groups.`